### PR TITLE
Blog improvements

### PR DESCRIPTION
**Pull request by kesar**
fixes https://github.com/EveripediaNetwork/issues/issues/2595

### CODE REVIEW

File: src/pages/blog/[digest].tsx [hunk 1]
```diff
@@ -169,7 +169,11 @@ export const BlogPostPage = ({
 
 export const getStaticProps: GetStaticProps = async (context) => {
   const digest: string = context.params?.digest as string
-  const result = await store.dispatch(getEntry.initiate(digest))
+  const [result, translations, blogEntries] = await Promise.all([
```
Comment (kesar) [inline src/pages/blog/[digest].tsx:172]: lets remember to try to put together blocking calls together so we improve time of generation

File: src/pages/blog/index.tsx [hunk 2]
```diff
@@ -46,22 +46,18 @@ export const Blog = ({ blogEntries }: { blogEntries: BlogType[] }) => {
 }
 
 export const getStaticProps: GetStaticProps = async ({ locale }) => {
-  const blogEntries = await getBlogsFromAllAccounts()
-  await Promise.all(store.dispatch(ArweaveApi.util.getRunningQueriesThunk()))
-
-  blogEntries?.sort((a, b) => {
```
Comment (kesar) [inline src/pages/blog/index.tsx:52]: this sort looks better inside of the getBlogsFromAccount so the page compontent is smaller and more readable

File: src/utils/blog.utils.ts [hunk 3]
```diff
@@ -55,22 +55,29 @@ export const formatBlog = (blog: Blog, hasBody?: boolean) => {
...
-  let blogs: FormatedBlogType[] = []
+export const getBlogsFromAccounts = async () => {
   const accounts = [config?.blogAccount2, config?.blogAccount3]
 
-  // eslint-disable-next-line no-plusplus
-  for (let i = 0; i < accounts.length; i++) {
+  const blogPromises = accounts.map(async (account) => {
     // eslint-disable-next-line no-await-in-loop
-    let { data: entries } = await store.dispatch(getBlogs.initiate(accounts[i]))
+    const { data: entries } = await store.dispatch(getBlogs.initiate(account))
```
Comment (kesar) [inline src/utils/blog.utils.ts:63]: avoid loops w awaits instead create promises and do a promise all in the end, so everything runs in parallel . in this case its a 3x improvement

File: src/pages/blog/[digest].tsx (hunk 1 above)
Comment (Tolu-Mals) [reply src/pages/blog/[digest].tsx:172]: Alright sir

File: src/pages/blog/index.tsx (hunk 2 above)
Comment (Tolu-Mals) [reply src/pages/blog/index.tsx:52]: That's true

**Review by Tolu-Mals — APPROVED** [review]

**Pull request merged**